core/src/components/datetime/datetime.scss

```diff
@@ -267,19 +267,24 @@ ion-picker-column-internal {
   justify-content: space-between;
 }
 
-:host .calendar-action-buttons ion-item,
+:host .calendar-action-buttons .calendar-month-year-toggle {
+  background: transparent;
+}
+
 :host .calendar-action-buttons ion-button {
-  --background: translucent;
+  --background: transparent;
 }
 
-:host .calendar-action-buttons ion-item ion-label {
+:host .calendar-action-buttons .calendar-month-year-toggle ion-label {
```

Comment:
Actually, we're not using an `ion-label` here anymore, are we? It's just a regular `label`. I think this rule can be removed entirely.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/28443/commits/d2d74f2349675e03c1c6f11cf9e523579ec94dd0